TestAssets/TestPackages/dotnet-dependency-tool-invoker/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.DependencyInvoker
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            var dotnetParams = new DotnetBaseParams("dotnet-dependency-tool-invoker", "DotNet Dependency Tool Invoker", "Invokes tools declared as NuGet dependencies of a project");
+            
+            dotnetParams.Parse(args);
+            
+            if (string.IsNullOrEmpty(dotnetParams.Command))
+            {
+                Console.WriteLine("A command name must be provided");
+                
+                return;
+            }
+            
+            var projectContexts = 
+                CreateProjectContexts(dotnetParams.ProjectPath)
+                    .Where(p => dotnetParams.Framework == null ||
+                                dotnetParams.Framework.GetShortFolderName()
+                                .Equals(p.TargetFramework.GetShortFolderName()));
```

Comment:
NuGetFramework.Equals should give the same result as what you have above. Are these frameworks constructed manually or created with NuGetFramework.Parse?